packages/form-js-viewer/src/render/components/FormField.js

```diff
@@ -55,6 +59,18 @@ export default function FormField(props) {
     properties.disabled || field.disabled || false
   );
 
+  const onBlur = useCallback(() => {
+    if (viewerCommands) {
+      viewerCommands.updateFieldValidation(field, value);
+    }
+  }, [ viewerCommands, field, value ]);
+
+  useEffect(() => {
+    if (viewerCommands && initialValue) {
+      viewerCommands.updateFieldValidation(field, initialValue);
+    }
+  }, [ viewerCommands, field, initialValue ]);
```

Comment:
I'm wondering whether this should be configurable, so you can restore the legacy behavior if you want to. Something like (with a better name):

```js
const form = new Form({
  properties: {
    noValidationBlurChange: true
  }
});
```

Let's also add a note in the breaking changes in the Changelog already 👍

Reply:
Since this is kind of a standard behavior, I would say it's not necessarily something we should qualify as a breaking change. Personally my take is that we treat it as a UX improvement, and if someone gets back to us saying they preferred the old behavior, we implement a conditional on this.

Reply:
agree!